ui/components/app/permissions-connect-permission-list/permissions-connect-permission-list.js

```diff
@@ -10,15 +10,15 @@ export default function PermissionsConnectPermissionList({ permissions }) {
   // TODO:flask Fix Snap permission labels and support namespaced permissions
   const PERMISSION_TYPES = useMemo(() => {
     return {
-      eth_accounts: {
+      'eth_accounts': {
         leftIcon: 'fas fa-eye',
         label: t('eth_accounts'),
         rightIcon: null,
       },
       ///: BEGIN:ONLY_INCLUDE_IN(flask)
-      snap_confirm: {
+      'snap_confirm': {
         leftIcon: 'fas fa-eye',
-        label: 'snap_confirm',
+        label: t('snap_confirm'),
```

Comment:
Do we have a localization with the key `snap_confirm`?  These are generally camelcased.

Reply:
They're generally correct, but we will establish keys for all permissioned RPC methods we add, in the vein of `eth_accounts`: https://github.com/MetaMask/metamask-extension/blob/develop/app/_locales/en/messages.json/#L1041-L1044